passes/erreq/erreq.go

```diff
@@ -11,27 +12,47 @@ import (
 
 var Analyzer = &analysis.Analyzer{
 	Name: "erreq",
-	Doc:  Doc,
+	Doc:  "erreq finds binary ops and switch-case statements comparing errors",
 	Run:  run,
 	Requires: []*analysis.Analyzer{
 		inspect.Analyzer,
 	},
 }
 
-const Doc = "erreq finds binary expression which is comparing error"
-
 var errType = types.Universe.Lookup("error").Type().Underlying().(*types.Interface)
 
+var causeFuncs = []string{
+	"TestCauseFunc", // For test
```

Comment:
We can test with `xerrors.Unwrap`.
So please remove this one.

Reply:
done